api/v1alpha1/lvmcluster_types.go

```diff
@@ -143,6 +143,12 @@ type DeviceSelector struct {
 	ForceWipeDevicesAndDestroyAllData *bool `json:"forceWipeDevicesAndDestroyAllData,omitempty"`
 }
 
+type DevicePath string
```

Comment:
Why do we need this type now? This only has one method which returns d the path as a string

Reply:
Pretty unidiomatic, but I agree with you. This should protect from unintended usage.